e2e-tests/cypress/tests/integration/boards/create_board_spec.ts

```diff
@@ -47,6 +47,54 @@ describe('Create and delete board / card', () => {
             should('have.value', 'Testing');
     });
 
+    it('MM-T4275 Set up Board description', () => {
+        cy.visit('/boards');
+
+        // # Create an empty board and change tile to Testing
+        cy.findByText('Create an empty board').should('exist').click({force: true});
+        cy.get('.BoardComponent').should('exist');
+
+        // Change Title
+        cy.findByPlaceholderText('Untitled board').should('be.visible').wait(timeouts.HALF_SEC);
+
+        // * Assert that the title is changed to "testing"
+        cy.findByPlaceholderText('Untitled board').
+            clear().
+            type('Testing').
+            type('{enter}').
+            should('have.value', 'Testing');
+
+        // # "Add icon" and "Show description" options appear
+        cy.findByText('Add icon').should('exist').click({force: true});
+        cy.findByText('show description').should('exist').click({force: true});
+
+        // # Click on "Add a description" below the board title and type "for testing purposes only"
+        cy.findByText('Add a description...').should('be.visible').wait(timeouts.HALF_SEC);
+
+        // * Assert that the editable description should be visible
+        cy.findByText('Add a description...').should('be.visible');
+        cy.findByText('Add a description...').click({force: true});
+        cy.get('.description').
+            click().
+            get('.description .MarkdownEditorInput').
+            type('for testing purposes only');
```

Comment:
I'm still getting some flakiness in test with `for testing purposes only` not getting set as the description. A workaround I found is to click anywhere to give trigger and time for description to be saved. Let me know if it works for you.

```suggestion
            type('for testing purposes only');

        // # Click to other element to give some time for the description to be saved.
        cy.findByPlaceholderText('Untitled board').click();
```